src/status_im/ui/screens/activity_center/utils.cljs

```diff
@@ -0,0 +1,6 @@
+(ns status-im.ui.screens.activity-center.utils)
+
+(defn contact-name
+  [contact]
+  (or (get-in contact [:names :nickname])
+      (get-in contact [:names :three-words-name])))
```

Comment:
Can [this](https://github.com/status-im/status-mobile/commit/5e814b7a035ebdfe13fa23d52dae5d8ee7b62c87) PR be helpful here ?

Reply:
I think you're onto something! This piece of code is based on the original Activity Center code, but we can clearly see it doesn't handle the names as that PR, which does something like:

```clojure
(->> [nickname ens-name display-name three-words-name short-public-key]
     (remove string/blank?)
     (take 2))
```

I'm taking note of your comment and I'll try to investigate in the future and open an issue if necessary. Are you okay with that?

Reply:
Definitely